doc/content/gateways/lorix-one/lbs.md

```diff
@@ -19,20 +19,39 @@ In the list, select **LoRa Basic Station** and press apply.
 
 {{< figure src="../lorix-one-lora-settings-forwarder-change-bs.png" alt="LORIX One LoRa forwarder Basic Station selection" class="plain" >}}
 
-On the new page, scroll down to the **LoRaWAN Network Server** section and enable it.
+## LNS
+
+Scroll down to the **Configuration** section and enable the **LoRaWAN Network Server**.
 
 {{< figure src="../lorix-one-lora-settings-bs.png" alt="LORIX One LoRa forwarder Basic Station LNS" >}}
 
-Follow the instructions for [Connecting LNS]({{< ref "/gateways/lora-basics-station/lns" >}}).
+To connect the gateway via LNS protocol, follow the instructions for [Connecting LNS]({{< ref "/gateways/lora-basics-station/lns" >}}).
+
+- Configure the **Address** and **Port** with the server address of your deployment and LNS port `8887`
+
+- Enable the **Secured TLS connection** and upload the [LNS Server Certificate]({{< ref "/gateways/lora-basics-station/lns#lns-server-certificate--lns-trust" >}} ) as a **Server authentication certificate**
+
+- Enable the **Client authentication** and copy paste the [LNS Key File]({{< ref "/gateways/lora-basics-station/lns#lns-key-file" >}} )
 
-If using CUPS, following the instructions for [Connecting CUPS]({{< ref "/gateways/lora-basics-station/cups" >}}).
 
 {{< figure src="../lorix-one-lora-settings-bs-lns.png" alt="LORIX One LoRa forwarder Basic Station LNS" class="plain" >}}
```

Comment:
It would be nice if this server address wasn't blurred

Reply:
@nejraselimovic 

We have taken the images from the `Lorixone` product documentation. Ref: https://iot.wifx.net/docs/lorix-os/latest/user-s-guide/lorawan/lorawan-networks/the-things-network/the-things-network-v3-since-2021

The `server address` in the image pointed to `eu1.cloud.thethings.network`. As `The Things Stack docs cover all the distributions`, we have blurred the server address in the image.

Note: We don't have a physical Gateway with us.

Reply:
Yeah, but the text says "Configure the **Address** with the server address of your deployment...", which explains that it doesn't only work for eu1 community deployment. People like to see the example (and most of them will use eu1 community deployment anyways), so please take back the original image without blurring the server address